tasks/runtime.yml

```diff
@@ -71,12 +71,12 @@
   service:
     name: munge
     enabled: "{{ openhpc_slurm_service_enabled | bool }}"
-    state: "{{ 'started' if openhpc_slurm_service_enabled | bool else 'stopped' }}"
+    state: "{{ 'started' if openhpc_slurm_service_started | bool else 'stopped' }}"
 
 # In case the service isn't running and the config hasn't changed to trigger
 # the handler, ensure it's running here.
 - name: Ensure Slurm services are running
```

Comment:
Name's not going to be correct any more (bit confusing at least)

Reply:
done